src/IO/AssimpLoader/AssimpGeometryDataLoader.hpp

```diff
@@ -70,6 +70,9 @@ class RA_IO_API AssimpGeometryDataLoader : public Core::Asset::DataLoader<Core::
     void fetchType( const aiMesh& mesh, Core::Asset::GeometryData& data ) const;
 
     /// Fill \p data with the vertices from \p mesh.
+    /// @warning This function lock the vertices attrib, you must unlock the attrib with unlock()
+    /// using getAttribManager() and getAttribBase( const Index &idx ) or by using an getUnlocker().
+    /// @see Unlocker
```

Comment:
Why don't unlock in fetchVertices ?

Reply:
It's because we use fetchVertices(), fetchTangent(), fetchBitangent(), etc ... In AssimpGeometryDataLoader::loadMeshAttrib (line 65 of AssimpGeometryDataLoader.cpp) And we can use getUnlocker and the beginning of the function to unlock all of them just with one row. But in fact I forgot to unlock it, so I will change that.

Reply:
IMHO better to lock/unlock in each function

Reply:
Ok, @MathiasPaulin probably don't agree with you (it was my first idea but we choose to do that with unlocker)

Reply:
Fine, wait for @MathiasPaulin comment. But I do not see why ask the user to unlock here.

Reply:
As fetchVertices() lock the vertices and copy all the data in the locked buffer, i think it should unlock also. The use of unlocker is helpful in other functions were acquiring a reference to the locked buffer, filling the locked buffer and unlocking it is not done in "one block" as done in fetchvertices(), fetchedges(), and so on.